Dev_1.java

```diff
@@ -0,0 +1,15 @@
+public class Dev_1 {
+    public static void main(String[] args){
+        int temp;
+        for(int i = 0; i <= 100; i++) {
```

Comment:
it's relly hard to find the mistake through the whole style document. Pleeease, give a hint